### PR TITLE
Added cloud setup instructions

### PR DESCRIPTION
**Pull request by o1brad**
Added GPU AMI setup instructions and updated the main tutorial and fast setup mode.

### CODE REVIEW

File: docs/index.markdown [hunk 1]
```diff
@@ -74,13 +104,13 @@ This will create three binaries. We won't go through them in detail right now bu
...
-./generate_parameters
-time ./main MNT4753 compute MNT4753-parameters MNT4753-input outputs
+./generate_parameters fast
+time ./main MNT4753 compute MNT4753-parameters MNT4753-input MNT4753-output
 ```
 This will save your program's output to the file `./outputs`.
 
 Now let's make a change to the implementation and re-compile.
-In `libsnark/main.cpp`, find the line 
+In `libsnark/main.cu`, find the line
```
Comment (o1brad) [inline docs/index.markdown]: @cmr is this the right file?

File: spec-lang/main.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+body {
```
Comment (imeckler) [inline spec-lang/main.css]: Did you add this by accident?

File: spec-lang/ladder.svg [hunk 3]
```diff
@@ -0,0 +1,417 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
Comment (imeckler) [inline spec-lang/ladder.svg]: Did you add this by accident?

File: spec-lang/dune-project [hunk 4]
```diff
@@ -0,0 +1 @@
+(lang dune 1.6)
```
Comment (imeckler) [inline spec-lang/dune-project]: Don't include this file -- it's a build artifact also. Maybe add it to the gitignore?

File: spec-lang/main.css (hunk 2 above)
Comment (o1brad) [reply spec-lang/main.css]: yes, removed it

File: spec-lang/ladder.svg (hunk 3 above)
Comment (o1brad) [reply spec-lang/ladder.svg]: yep

File: spec-lang/dune-project (hunk 4 above)
Comment (o1brad) [reply spec-lang/dune-project]: okay removed it

**Pull request merged**